### PR TITLE
Travis Updates

### PR DESCRIPTION
**Pull request by maclover7**
Two improvements for Travis builds:

- Travis will now alert the 'bundler-api' slack room about builds, in the same way it currently does in the 'bundler' room.
- Travis will deploy to the 'bundler-api-staging' Heroku app upon a green build.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -17,8 +17,11 @@ before_script:
...
-    rooms:
-      - secure: "B4ev1d4ZH0Xro49xIunvZNAF2LR0hDgvfpzYMcqUlLNtY5YSmUHbyroVU60w\n10pgYkEhH3DXzHGSYLoYlxGC7foFCZhTkf9JaNgIoSnQmFet+7C66wEkskoY\nwa9n1V3WGic5ZFVBnrZ1IFM6D3hLlAfQdwnwC5GWSrSS9B0jjdI="
+  slack:
+    secure: iIhKCxayvPZbgASh9iQsPgUo3T/bMGDC8gJd4EO+LsIrzCVHHRV8/lx+0BisW4RaxYKFo1m2QOzJhSKYv/s4XpfZo7iyM8igRS6Ex+K7ZZtSSAmdq9Da7+45NE/q1eTILMcSeNmbfiuj8tX1nrU+G+w/+N6VcL6MoncxCJi+EAs=
+
+deploy:
+  provider: heroku
+  api_key:
+    secure: CqqIlZu3vkiQoVgrvwwYQPJt/od8CeuicY8BzWF3fN2dIb7hAdvJTr/0neeve3FFoboWPezrylrzMlp8bXxKYI7zM67Q2RitoU72tmrKJNV27Si1L7hVh1eTLVCdYOJCFcxE1sntEBkiXhz6qxK2C/XS/1TwfT5NEsMywqXWl+8=
+  app: bundler-api-staging
```
Comment (dwradcliffe) [inline .travis.yml]: nitpick: can you add a newline at the end?

**Comment by indirect** [comment]
Sounds good. Thanks!

File: .travis.yml (hunk 1 above)
Comment (maclover7) [inline .travis.yml]: Committed.

**Comment by maclover7** [comment]
Green build.

**Comment by andremedeiros** [comment]
Thanks, Jon! :+1:

**Comment by segiddins** [comment]
Hm, this is an issue since Andre and I have been using staging to test out new index stuff

**Comment by indirect** [comment]
@segiddins we can deploy manually to staging on demand or just fork the staging app for new index testing

**Pull request merged**